exchange/utils.go

```diff
@@ -49,13 +49,17 @@ func cleanOpenRTBRequests(ctx context.Context,
 		COPPA: orig.Regs != nil && orig.Regs.COPPA == 1,
 	}
 
+	var ccpaPolicy ccpa.Policy
 	if enforceCCPA {
-		ccpaPolicy, _ := ccpa.ReadPolicy(orig)
-		privacyEnforcement.CCPA = ccpaPolicy.ShouldEnforce()
+		ccpaPolicy, _ = ccpa.ReadPolicy(orig)
```

Comment:
Yes. We would want to check the error, but it's not a fatal error. It would be a warning that CCPA is being ignored. We don't have the "plumbing" in place right now to pass the warning up the chain. There's an issue internal to the Xandr team to add that in.

Reply:
To be clear, the error only occurs when the json is malformed. Unlikely since there is validation earlier in the pipeline.